lib/internal/navigator.js

```diff
@@ -102,27 +102,31 @@ class Navigator {
    * @return {string}
    */
   get language() {
+    this.#language ??= Intl?.Collator().resolvedOptions().locale || 'en-US';
     return this.#language;
   }
 
   /**
    * @return {Array<string>}
    */
   get languages() {
+    this.#languages ??= ObjectFreeze([this.language]);
```

Comment:
Review note: note the difference here. The original was accessing the private `#language`, while this one exercises the `get language()` getter so that `#language` gets cached.

Reply:
I had missed this in the first pass but apparently we don't have a test that actually verifies that `languages` matches `language`? Will have to look at that with a different PR.